File: 2_semester/Laba_6/task1_2.py
N = 3
M = 3;
N, M = map(int,input().split())
temp = [0 for i in range(M)]
vec = []

for i in range(N):
	vec.append(list(map(int, input().split())))

max_index = vec.index(max(vec))
min_index = vec.index(min(vec))
vec[max_index], vec[min_index] = vec[min_index], vec[max_index]

for vect in vec:
	for item in vect:
		print(item, end = ' ')
	print()

File: 2_semester/Laba_6/task1_2.cpp
#include <iostream>
#include <vector>
#include <limits.h>
using namespace std;

int main(){

	int N = 3, M = 3;
	cin >> N >> M;
	vector<int> temp(M);
	vector<vector<int>> vec (N);

	for (auto &vect : vec){
		for (auto &item : temp)
			cin >> item;
		vect = temp;
	}
	
	int64_t max= INT_MIN, min = INT_MAX;
	int max_index, min_index;
	for (int i = 0; i < N; i++){
		for (auto item : vec[i]){
			if (item > max){
				max = item;
				max_index = i;
			}
			if (item < min){
				min = item;
				min_index = i;
			}
		}
	}

	vec[max_index].swap(vec[min_index]);

	for (auto vect : vec){
		for (auto item : vect)
			cout << item << ' ';
		cout << endl;
	}
}